domain_utils/domain_utils.py

```diff
@@ -264,3 +275,22 @@ def get_scheme(url, no_scheme=NO_SCHEME):
         return scheme
     else:
         return no_scheme
+
+
+def get_port(url):
```

Comment:
and then the extractor should be passed along to the new private method.

Reply:
you don't need to add a test for this in this PR, because I just wrote a test for the same thing in #22 which i will repurpose to test this new private method. (we'll get this PR merged before #22).